clang/unittests/Format/FormatTestTableGen.cpp

```diff
@@ -101,6 +101,9 @@ TEST_F(FormatTestTableGen, BangOperators) {
                "                                  \"zerozero\",\n"
                "                                  true:  // default\n"
                "                                  \"positivepositive\");\n"
+               "  let Foo = !cond(\n"
```

Comment:
How about describing in name?

```suggestion
               "  let CondOpeCommentAfterLParen = !cond(\n"
```